Material.Blazor/Components/DebouncedTextField/MBDebouncedTextField.razor

```diff
@@ -3,7 +3,10 @@
 @inherits InputComponent<string>
 
 
-<MBTextField @attributes="AttributesToSplat()"
+<MBTextField class="@(@class)"
```

Comment:
Here we don't need to consider `MappedClassesAndUserClasses`, since `MBDebouncedTextField` doesn't add any more CSS classes in addition to `MBTextField`.